Source/Form/Gen4/Static4.cpp

```diff
@@ -183,9 +183,20 @@ void Static4::generatorCategoryIndexChanged(int index)
         {
             if ((currentProfile->getVersion() & templates[i].getVersion()) != Game::None)
             {
-                ui->comboBoxGeneratorPokemon->addItem(
-                    QString::fromStdString(Translator::getSpecie(templates[i].getSpecie(), templates[i].getForm())),
-                    QVariant::fromValue(i));
+
+                if (templates[i].getShiny() == Shiny::Never)
+                {
+                    ui->comboBoxGeneratorPokemon->addItem(
+                        QString::fromStdString(Translator::getSpecie(templates[i].getSpecie(), templates[i].getForm()))
+                            + " (" + tr("Shiny Locked") + ")",
+                        QVariant::fromValue(i));
+                }
+                else
+                {
+                    ui->comboBoxGeneratorPokemon->addItem(
+                        QString::fromStdString(Translator::getSpecie(templates[i].getSpecie(), templates[i].getForm())),
+                        QVariant::fromValue(i));
+                }
```

Comment:
Do we want to do this or just add a shiny combobox that gets filled out by the template like we have in the other windows?

Reply:
Can't we have both? Or you prefer having only one of the two?

Imo, i would keep both. But if you want only one of the two, the shiny combobox would be better because "Shiny Locked" text is being cut.

Reply:
Done